job-templates/kubernetes_release.json

```diff
@@ -3,7 +3,8 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.13"
+      "orchestratorRelease": "1.14",
+      "azureCNIVersion": "v1.0.17"
```

Comment:
`FATA[0000] error loading API model in generateCmd: error parsing the api model: Unknown JSON tag azureCNIVersion`

Reply:
Yes. See comment above. I just had a test job fail a minute ago with the exact same thing. Changing.

Reply:
I'll push a fix

Reply:
```
      "kubernetesConfig": {
          "azureCNIVersion": "v1.0.17"
      }
```

That seems to work.